src/main/java/gift/service/KakaoOAuthService.java

```diff
@@ -0,0 +1,93 @@
+package gift.service;
+
+import gift.dto.api.KakaoTokenResponseDto;
+import gift.dto.api.KakaoUserResponseDto;
+import gift.exception.KakaoOAuthException;
+import io.netty.channel.ChannelOption;
+import java.time.Duration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatusCode;
+import org.springframework.http.MediaType;
+import org.springframework.http.client.reactive.ReactorClientHttpConnector;
+import org.springframework.stereotype.Service;
+import org.springframework.web.reactive.function.BodyInserters;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Mono;
+import reactor.netty.http.client.HttpClient;
+
+@Service
+public class KakaoOAuthService {
+
+    @Value("${kakao.clientId}")
```

Comment:
```suggestion
    @Value("${kakao.client-id}")
```
여기도 일관성있게 이렇게 하면 좋겠네요!

Reply:
이 부분도 마찬가지로 일관성 있게 수정하도록 하겠습니다!